src/components/ViewedByCustomer/ViewedByCustomer.ts

```diff
@@ -55,7 +55,9 @@ export class ViewedByCustomer extends Component {
     ) {
         super(element, ViewedByCustomer.ID, bindings);
         if (this.root.getElementsByClassName(Component.computeCssClassNameForType(UserActions.ID)).length === 0) {
-            throw new Error('No UserActions component found on the page template.');
+            this.logger.warn(
+                'ViewedByCustomer component has been detected without a UserAction components, you may encounter issues with the former.'
```

Comment:
Doc approved?

Reply:
The ViewedByCustomer component has been detected without a UserActions component. You may encounter issues with the former.
(Then doc approved :D)

Reply:
Done!